spanner/spanner_snippets/snippet.go

```diff
@@ -906,6 +916,264 @@ func queryWithTimestamp(ctx context.Context, w io.Writer, client *spanner.Client
 
 // [END spanner_query_data_with_timestamp_column]
 
+// [START spanner_dml_standard_insert]
+
+func insertUsingDML(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			rowCount, err := txn.Update(ctx, spanner.Statement{
```

Comment:
Maybe declare the `spanner.Statement` before this line? Getting a little difficult to follow with the indentation.